cmd/clairctl/docker/push.go

```diff
@@ -0,0 +1,68 @@
+package docker
```

Comment:
We could add Push in analyze if layer is not found yes. I like to keep the `push` command to allow user to push it on regular basis, without getting directly the analyze